pom.xml

```
@@ -11,7 +11,7 @@
     <properties>
         <swagger-annotations-version>1.5.22</swagger-annotations-version>
         <jersey-version>2.27</jersey-version>
-        <jackson-version>2.8.9</jackson-version>
+        <jackson-version>2.11.0</jackson-version>
```

does this fix the issue
```suggestion
        <jackson-version>2.11.0</jackson-version>
```